modules/dxp/apps/segments/segments-experiment-web/src/main/java/com/liferay/segments/experiment/web/internal/product/navigation/control/menu/SegmentsExperimentProductNavigationControlMenuEntry.java

```diff
@@ -342,6 +337,9 @@ private void _processBodyBottomTagBody(PageContext pageContext) {
 	private static final String _ICON_TMPL_CONTENT = StringUtil.read(
 		SegmentsExperimentProductNavigationControlMenuEntry.class, "icon.tmpl");
 
+	private static final String _PANEL_STATE_SESSION_KEY =
```

Comment:
@YurenaC Brian changed the name of the key in https://github.com/brianchandotcom/liferay-portal/commit/b32e51e76ab7ca23e7073d8c493567fbf8394f24. We should call it the same.

Reply:
Absolutely right, on my way!